ocrd_validators/ocrd_validators/processing_server_config.schema.yml

```diff
@@ -58,6 +58,7 @@ properties:
         - address
         - username
         - workers
```

Comment:
I think it would be good to allow only workers or only servers. For that I suggest replacing the following:
```
        - workers
        - servers
      oneOf:
        - required:
            - password
        - required:
            - path_to_privkey
```
with this
```
      allOf:
        - oneOf:
          - required:
              - password
          - required:
              - path_to_privkey
        - oneOf:
          - required:
              - workers
          - required:
              - servers
```

Reply:
Agree. As it is currently, it restricts starting only workers or only servers. However, I think it should be `anyOf` instead of `oneOf`. I would still keep it flexible and allow deploying workers and servers simultaneously.

Reply:
Yes, right, `anyOf` is better

Reply:
Resolved. Now it's possible to start only workers or standalone servers or both.